misk/src/main/kotlin/misk/tracing/TracingModule.kt

```diff
@@ -5,12 +5,12 @@ import misk.tracing.backends.jaeger.JaegerBackendModule
 
 class TracingModule(val config: TracingConfig) : KAbstractModule() {
   override fun configure() {
-    if (config.tracer.equals("jaeger", true)) {
+    if (config.backends.jaeger != null) {
       install(JaegerBackendModule(config.backends.jaeger))
       return
     }
 
-    throw IllegalArgumentException("No backend for '${config.tracer}' tracer. Ensure that " +
-        "tracer name is spelled correctly and that a backend module exists for that tracer.")
+    throw IllegalStateException("No backends configured for tracing. Please update your yaml "
```

Comment:
basically the tab vs. whitespace debate of exception throwing. I see you've chosen your religion

Reply:
😬